FunderMaps.Webservice.Core/FunderMaps.Webservice.Core.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
```

Comment:
For all future projects just target .NET Core LTS. Its unlikely we'll need to port some back to .NET Framework